main.ts

```diff
@@ -0,0 +1,381 @@
+enum RcbVerticalDirection {
+    //% block="up" blockId="rollerCoasterBuilderUp"
+    Up,
+    //% block="down" blockId="rollerCoasterBuilderDown"
```

Comment:
Interesting idea, but I don't think it'd add much value in this case. This is used for ramps and spirals, both of which don't really make sense without vertical change in one direction or another (ramp with no height is just a straight line, which has a separate block; spiral with no height is just a loop, which would break the roller coaster)